sdk/log/logtest/builder.go

```diff
@@ -0,0 +1,131 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package logtest // import "go.opentelemetry.io/otel/sdk/log/logtest"
+
+import (
+	"context"
+	"slices"
+	"time"
+
+	"go.opentelemetry.io/otel/log"
+	"go.opentelemetry.io/otel/sdk/instrumentation"
+	sdklog "go.opentelemetry.io/otel/sdk/log"
+	"go.opentelemetry.io/otel/sdk/resource"
+)
+
+// TODO: comment.
+type Builder struct {
+	timestamp time.Time
+	// TODO: implement all fields
+	// observedTimestamp time.Time
+	// severity          log.Severity
+	// severityText      string
+	// body              log.Value
+	attrs []log.KeyValue
+	// traceID           trace.TraceID
+	// spanID            trace.SpanID
+	// traceFlags        trace.TraceFlags
+
+	resource *resource.Resource
+	scope    instrumentation.Scope
+
+	dropped int
+}
+
+// TODO: comment.
+func (b Builder) Record() sdklog.Record {
```

Comment:
```suggestion
func (b Builder) Build() sdklog.Record {
```

Builder builds

Reply:
That is why I decided to name it Record to follow the strings.Builder